### DISCUSSION TITLE
Сколько требуется свободного места в памяти роутера для установки?

### DISCUSSION THREAD

**Discussion by lsahnicne** [bol-van/zapret#279] [Q&A] [answered]
В [quick_start.txt](https://github.com/bol-van/zapret/blob/master/docs/quick_start.txt) написано, что для установки самого zapret и необходимых дополнительных пакетов требуется около 1-2 Mb свободного места. Однако следующим же шагом предлагается скачать и распаковать архив проекта, который в распакованном виде занимает аж 7,6 Mb.
Подразумевается, что после установки файлы из распакованного архива проекта можно удалить или строка где сказано про 1-2 Mb безнадёжно устарела?

**Comment by bol-van** [comment, marked as answer]
/tmp это рам диск
оперативки хватит
инсталятор скопирует около 600 кб
остальное пакеты
